Gemfile

```diff
@@ -27,6 +27,13 @@ gem 'jbuilder', '~> 2.0'
 # bundle exec rake doc:rails generates the API under doc/api.
 gem 'sdoc', '~> 0.4.0', group: :doc
 
+gem 'will_paginate', '~> 3.0.6'
```

Comment:
will_paginate is not bad, actually its written by my friends, however i recomend you to have a look at the Kaminari gem as well